src/components/GroupAll/groupFilter/LanguageSelectBox.tsx

```diff
@@ -0,0 +1,325 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { IcArrowBottomGray, IcArrowTopGray, IcFilter } from '../../../assets';
+import CommonHashTag from '../../../common/CommonHashTag';
+import { LanguageSectionProps } from '../../../types/GroupCreate/GroupCreateType';
+import PageLayout from '../../PageLayout/PageLayout';
+import RangeSliderFilter from './RangeSlidrFilter';
+
+const ALL_TAG = 'ALL';
+const firstRowTags = ['Python', 'Java', 'JavaScript', 'C++', 'C', 'C#'];
+const secondRowTags = ['Kotlin', 'Swift', 'Ruby', 'Scala', 'Go'];
+
+const LanguageSelectBox = ({
+  selectedTags,
+  setSelectedTags,
+}: LanguageSectionProps) => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  // 태그와 슬라이더의 상태를 별도로 관리
+  const [tempMinValue, setTempMinValue] = useState<number>(1);
+  const [tempMaxValue, setTempMaxValue] = useState<number>(50);
```

Comment:
P2: 아래처럼 할 수 있을 것 같아요!

```suggestion
  const [tempValue, setTempValue] = useState({
    min: 0,
    max: 50
  });
```

Reply:
네! 수정하겠습니당